src/main/java/com/ecore/roles/service/impl/UsersServiceImpl.java

```diff
@@ -3,26 +3,24 @@
 import com.ecore.roles.client.UsersClient;
 import com.ecore.roles.client.model.User;
 import com.ecore.roles.service.UsersService;
-import org.springframework.beans.factory.annotation.Autowired;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.UUID;
 
 @Service
+@RequiredArgsConstructor
 public class UsersServiceImpl implements UsersService {
 
     private final UsersClient usersClient;
 
-    @Autowired
```

Comment:
This constructor can be replaced with @RequiredArgsConstructor lombok annotation in order to become the code cleaner